File: Loiane_Training_Java/Java_Basico/Exer Aula 18_19/Ex28.java
public class Ex28 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
        int[] b = new int[10];

        for (int i = 0; i < a.length; i++) {
            b[i] = a[a.length - i - 1];
            System.out.println(a[i] + "\t" + b[i]);
        }
    }
}

File: loiane-training-csharp/procedural-poo/d/Ex28.cs
using System;

class Ex28
{
    static void Main(string[] args)
    {
        Console.WriteLine("Ola colecionador de CD\'s, voce colecionou quantos CD\'s ate agora? ");
        int quantidadeDaColecao = int.Parse(Console.ReadLine());

        int totalInvestido = 0;
        for (int i = 0; i < quantidadeDaColecao; i++)
        {
            Console.WriteLine("Voce lembra quanto gastou no {0} CD?", i + 1);
            int valorDoCD = int.Parse(Console.ReadLine());

            totalInvestido += valorDoCD;
        }

        double mediaUnitariaDosCDs = totalInvestido / quantidadeDaColecao;

        Console.WriteLine("Total investido: {0}", totalInvestido);
        Console.WriteLine("Media de gasto de cada CD: R${0}", mediaUnitariaDosCDs);
    }
}